lib/govuk_publishing_components/presenters/machine_readable/government_service_schema.rb

```diff
@@ -0,0 +1,73 @@
+module GovukPublishingComponents
+  module Presenters
+    class GovernmentServiceSchema
+      attr_reader :page
+
+      def initialize(page)
+        @page = page
+      end
+
+      def structured_data
+        # http://schema.org/GovernmentService
+        {
+          "@context" => "http://schema.org",
+          "@type" => "GovernmentService",
+          "name" => page.title,
+          "serviceType" => page.description,
+          "description" => description,
```

Comment:
In our other schemas this is usually the `page.description` - possibly best to stick with that unless we have a strong reason not to?

Reply:
Also, this has raised a good point, that the Article schema doesn't seem to include the page body for transactions in it (because of the weird properties like `more_information` that you've seen).  We could probably do that there and keep this more simple for now?

Reply:
To confirm, this is getting rid of `serviceType` for now and using our `description` in the `description` property? I think this might work better actually, as `serviceType` is just a categorisation of the service.